ModernFlyouts/Properties/Strings.pl.resx

```diff
@@ -135,31 +135,31 @@
     <value>Włączone</value>
   </data>
   <data name="Settings.FlyoutBackgroundOpacity" xml:space="preserve">
-    <value>Przezroczystość okienka</value>
+    <value>Krycie w tle wysu podczas wysuń</value>
   </data>
   <data name="Settings.FlyoutDefaultPosition" xml:space="preserve">
     <value>Domyślna pozycja wysuwania (dla przycisku Wyrównaj)</value>
   </data>
   <data name="Settings.FlyoutDefaultPositionDescription" xml:space="preserve">
     <value>Wpisz pozycję w formacie "X,Y" (np. 50,60)</value>
   </data>
-   <data name="Settings.FlyoutTopBar" xml:space="preserve">
```

Comment:
Did you revert because of an error @Samuel12321 (also sorry for mass pinging)? Because I don't know if indentation is actually critical in .resx files...